packages/blockchain-utils-validation/src/blockchains/cosmos.ts

```diff
@@ -0,0 +1,59 @@
+import { AccountID } from "caip";
+import { Tx, SignMeta, verifyTx } from '@tendermint/sig';
+import { BlockchainHandler } from "../blockchain-handler";
+import type { LinkProof } from "@ceramicnetwork/blockchain-utils-linking";
+import * as uint8arrays from "uint8arrays";
+
+const namespace = "cosmos";
+
+const stringEncode = (str: string): string => uint8arrays.toString(uint8arrays.fromString(str), 'base64pad');
+const stringDecode = (str: string): string => uint8arrays.toString(uint8arrays.fromString(str, 'base64pad'));
+
+// return data in the cosmos unsigned transaction format
+function asTransaction(address: string, message: string): Tx {
```

Comment:
Use the one exported from linking package, please.

Would be pity, if someone forgets to change this in either linking or validation packages.